clang-hybrid/memcpy.c

```diff
@@ -40,7 +40,7 @@ __FBSDID("$FreeBSD$");
 #include <stdint.h>
 #else
 #ifndef _VADDR_T_DECLARED
```

Comment:
Macro needs changing surely?

Reply:
Oops, I did that in CheriBSD and failed to bring it over.